src/TritonRoute/src/dr/FlexGridGraph_maze.cpp

```diff
@@ -802,9 +802,21 @@ frMIdx FlexGridGraph::getUpperBoundIndex(const frVector<frCoord>& tracks,
       ||  // comment out for non-buffer enablement
       currGrid.getLastDir() == dir) {
     return false;
-  } else {
-    return true;
+  } 
+  if (ndr_) {
+      frCoord hw = (frCoord)design_->getTech()->getLayer(getLayerNum(currGrid.z()))->getWidth();
```

Comment:
What does hw stand for in this context?

Reply:
hw is often half-width (eg line 809) but we should use a better name.  However in this line it isn't a half-width but a full width which is more confusing.

Reply:
Yes its ultimate purpose is to be half width. Considering this is done 2 lines bellow, I thought it would be ok to do so, but I will change it now.